test-cypress/integration/repository/repositories.spec.js

```diff
@@ -628,7 +628,64 @@ describe('Repositories', () => {
         testOntopConfigurationElementsVisibility('Lenses file', '#lensesFile');
         testOntopConfigurationElementsVisibility('Ontology file', '#owlFile');
         //verify driver download url
-        compareDriverDownloadUrl('https://www.ibm.com/support/pages/db2-jdbc-driver-versions-and-downloads');
+        compareDriverDownloadUrl('https://www.google.com/search?q=db2+jdbc+download');
+
+        //Select Dremio driver type and verify elements and download url (if available)
+        selectDatabaseDriver('Dremio');
```

Comment:
When you add a default to the switch, also change this test, to check if going back to the other drivers would work ok.